src/test/javascript/spec/component/exercises/shared/result.spec.ts

```
@@ -126,6 +126,26 @@ describe('ResultComponent', () => {
+        const submission1: Submission = { id: 1 };
+        const result1: Result = { id: 1, submission: submission1, score: 0.8, assessmentType: AssessmentType.AUTOMATIC_ATHENA, successful: true };
+        const result2: Result = { id: 2 };
+        const participation1 = cloneDeep(modelingParticipation);
+        participation1.results = [result1, result2];
+        component.participation = participation1;
+        component.showUngradedResults = true;
+
+        fixture.detectChanges();
+
+        expect(component.result).toEqual(result1);
+        expect(component.result!.participation).toEqual(participation1);
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)